Adyen/client.py

```diff
@@ -171,6 +172,14 @@ def _determine_checkout_url(self, platform, action):
             action = "payments/details"
         if action == "paymentsResult":
             action = "payments/result"
+        if action == "paymentsCancelsWithoutReference":
+            action = "payments/cancels"
```

Comment:
are you referring to this endpoint? https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/cancels
it should be just `/cancels`? without the `payments` prefix. Maybe we can call the action just `cancels` if this is the case.